gemma-core/src/main/java/ubic/gemma/persistence/service/expression/experiment/ExpressionExperimentDaoImpl.java

```diff
@@ -77,14 +78,14 @@
 
     private static final int BATCH_SIZE = 1000;
 
-    private static final String
-            CHARACTERISTIC_ALIAS = CharacteristicDao.OBJECT_ALIAS,
-            BIO_MATERIAL_CHARACTERISTIC_ALIAS = "bmc",
-            FACTOR_VALUE_CHARACTERISTIC_ALIAS = "fvc",
-            ALL_CHARACTERISTIC_ALIAS = "ac",
-            BIO_ASSAY_ALIAS = BioAssayDao.OBJECT_ALIAS,
-            TAXON_ALIAS = TaxonDao.OBJECT_ALIAS,
-            ARRAY_DESIGN_ALIAS = ArrayDesignDao.OBJECT_ALIAS;
+    static final String
+            CHARACTERISTIC_ALIAS = CharacteristicDao.OBJECT_ALIAS;
+    static final String BIO_MATERIAL_CHARACTERISTIC_ALIAS = "bmc";
+    static final String FACTOR_VALUE_CHARACTERISTIC_ALIAS = "fvc";
+    static final String ALL_CHARACTERISTIC_ALIAS = "ac";
```

Comment:
This shouldn't be necessary, but there's no way around for detecting which `Filter` object applies to a characteristic.